src/docs/development/add-to-app/ios/add-flutter-screen.md

```diff
@@ -346,7 +622,7 @@ Connected views:
 
 Attach to specific isolates instead in two steps:
 
-1. Name the Flutter root isolate of interest in its Dart source.
+1- Name the Flutter root isolate of interest in its Dart source.
```

Comment:
Is `1-` the normal markdown way to number steps on the website?

Reply:
It's not, but the code highlighter makes the markdown parser crap out (it would have the wrong indent with the code block anyway)